### ISSUE TITLE
Input preprocessing for VGG

### ISSUE THREAD

**Issue by borisdayma** [matthias-wright/flaxmodels#5]
Hi,

In [the README](https://github.com/matthias-wright/flaxmodels/blob/main/flaxmodels/vgg/README.md#2-basic-usage), it is mentioned that input should be between 0 and 1.

In the [training code](https://github.com/matthias-wright/flaxmodels/blob/main/training/vgg/training.py), they seem to be between -1 and 1.

In the [torchvision doc](https://pytorch.org/vision/0.10/models.html#classification), they seem to be loaded between 0 and 1 and then normalized with
```
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                 std=[0.229, 0.224, 0.225])
```

Should they be preprocessed as per the torchvision docs?

**Comment by borisdayma** [comment]
Interestingly, in [lpips module](https://github.com/richzhang/PerceptualSimilarity/blob/c82f1ed07d9b5004b4b41ae7b07ec53468ba3e40/lpips/lpips.py#L113), they are just normalized to [-1, 1].

**Comment by matthias-wright** [comment]
Hi @borisdayma! 
When you are using the pretrained weights (pretrained='imagenet'), the input should be between 0 and 1.
The torchvision normalization is applied to the input in the `__call__` method so you don't have to do that, see [here](https://github.com/matthias-wright/flaxmodels/blob/main/flaxmodels/vgg/vgg.py#L91).
I am considering to add an additional argument `normalization` so that people can use the imagenet weights without using the torchvision normalization, what do you think about that?

For the training code I used the range [-1, 1] because that range has worked better for me in the past.

Yeah, for lpips the input is [-1, 1] and then the input is [normalized](
https://github.com/richzhang/PerceptualSimilarity/blob/c82f1ed07d9b5004b4b41ae7b07ec53468ba3e40/lpips/lpips.py#L147) using `mean=[-.030,-.088,-.188]` and `std=[.458,.448,.450]`.

**Comment by borisdayma** [comment]
Thanks, this is much clearer.
I think it's a good idea to add the argument `normalize` as a parameter.

**Comment by matthias-wright** [comment]
Thanks for the feedback! I added the argument:
```python
import flaxmodels as fm

vgg16 = fm.VGG16(output='logits', pretrained='imagenet', normalize=False)
```
This way the imagenet weights are used but the images are not normalized internally.
